front/app/containers/IdeasShowPage/IdeaShowPageTopBar.tsx

```diff
@@ -123,14 +123,11 @@ const IdeaShowPageTopBar = ({
     }
   }, [goBack, deselectIdeaOnMap, project]);
 
-  if (!phase) return null;
```

Comment:
Making sure this bar renders even without a phase, in which case only the go back button will render.

Reply:
I think it makes sense to remove this. Even if we don't have a, the go back button can be there.